src/VirtoCommerce.CustomerSegmentsModule.Core/Models/IMemberSearchCriteriaBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using VirtoCommerce.CustomerModule.Core.Model.Search;
+
+namespace VirtoCommerce.CustomerSegmentsModule.Core.Models
+{
+    public interface IMemberSearchCriteriaBuilder
+    {
+        MembersSearchCriteria Build();
```

Comment:
Better to do not include all fluent syntax into a interface, make them as extension methods instead